packages/react-native/template/README.md

```diff
@@ -0,0 +1,76 @@
+This is a [**React Native**](https://reactnative.dev) project bootstrapped with [`npx react-native@latest init`](https://github.com/react-native-community/cli/tree/main#creating-a-new-react-native-project).
+
+# Getting Started
+
+>**Note:** Make sure you have completed the [React Native - Environment Setup](https://reactnative.dev/docs/environment-setup) instructions till [Creating a new application](https://reactnative.dev/docs/environment-setup#:~:text=Creating%20a%20new%20application) step, before proceeding.
```

Comment:
This anchor does not seem to work unless you have already visited the website and clicked the CLI tab. The text is not found when the default Expo tab is selected. Can be tested with new private browser.